tests/testcontainers-env/src/main/resources/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM quay.io/alfresco/alfresco-content-repository:latest
+
+ADD https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.30/mysql-connector-java-8.0.30.jar /usr/local/tomcat/lib/mysql-connector-java-8.0.30.jar
```

Comment:
A bit of description in comment response please. I see that it takes the latest repository image (generated by maven during tests I suppose) and adds on the jar driver for mysql database, that's it here, right?